### PR TITLE
Notable changes for v2.0.5-1

### PR DESCRIPTION
**Pull request by Eirik0**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -4,3 +4,11 @@ release-notes at release time)
 Notable changes
 ===============
 
+
+Sprout to Sapling Migration Tool Fixes
+--------------------------------------
+This release includes fixes to the Sprout to Sapling Migration Tool found in
+testing. We resolved an issue which would cause the zcash daemon to crash if
+calling the RPC `z_getmigrationstatus` while a wallet's migration transactions
+are in the mempool. We have also added logic to prevent a race condition which
```
Comment (bitcartel) [inline doc/release-notes.md]: I don't think this last sentence is needed, and as worded, is not clear what might have been fixed.

**Comment by Eirik0** [comment]
Squashed.

**Comment by bitcartel** [comment]
@zkbot r+

**Comment by Eirik0** [comment]
@zkbot retry

File: doc/release-notes.md [hunk 2]
```diff
@@ -4,3 +4,53 @@ release-notes at release time)
...
+In the 2.0.4 release the consensus rules were changed on testnet to enforce a
+consensus rule which marks blocks as invalid if they would lead to a turnstile
+violation in the Sprout or Shielded value pools.
+**This release enforces the consensus rule change on mainnet**
+
+The motivations and deployment details can be found in the accompanying
+[ZIP draft](https://github.com/zcash/zips/pull/210) and
+[PR 3968](https://github.com/zcash/zcash/pull/3968).
+
+Developers can use a new experimental feature `-developersetpoolsizezero` to test
```
Comment (ioptio) [inline doc/release-notes.md]: Do they also have to include `-experimentalfeatures`? If so, let's clarify.

Comment (LarryRuane) [reply doc/release-notes.md]: @ioptio, yes, `-developersetpoolsizezero` does require `-experimentalfeatures`.

**Pull request merged**